cms/src/test/java/uk/nhs/digital/apispecs/SwaggerCodeGenOpenApiSpecificationJsonToHtmlConverterTest.java

```diff
@@ -72,6 +72,33 @@ public void getHtmlForSpec_throwsException_onCodeGenFailure() {
         // expectations set in 'given' are satisfied
     }
 
+    @Test
+    public void hideRequestHeaderWhenSectionIsEmpty() {
```

Comment:
The test data files used here are unnecessarily large for testing a small test case. This makes them hard to maintain and understand, and their size contributes to log limit being exceeded on Travis CI in the case of test failure, as their content is logged in the entirety.

The files should be cut down to the minimum necessary content required to exercise this test case.

Either the files should be cut

Reply:
Fixed